user-interface/src/lib/hooks/UseFeatureFlags.ts

```diff
@@ -12,10 +12,9 @@ export const PRIVILEGED_IDENTITY_MANAGEMENT = 'privileged-identity-management';
 
 export default function useFeatureFlags(): FeatureFlagSet {
   const config = getFeatureFlagConfiguration();
-  if (!config.clientId) return defaultFeatureFlags;
+  if (import.meta.env['CAMS_PA11Y'] === 'true') return testFeatureFlags;
```

Comment:
We might want something like this instead:
```suggestion
  if (import.meta.env['CAMS_PA11Y'] === 'true' || import.meta.env['DEFAULT_FEATURE_FLAGS_TRUE'] === 'true') return testFeatureFlags;
```

This would allow us to test locally easier while still being able to hit a locally-running backend. However, that would potentially cause issues with the flag values for the frontend and backend not matching. I'd probably just default to hitting Launch Darkly and changing the values through the portal there, which is a better test anyway, although potentially a worse developer experience.